README.md

```diff
@@ -32,6 +32,7 @@
 [Official chat](https://chat.reactos.org/) &bull;
 [Wiki](https://reactos.org/wiki/) &bull;
 [Forum](https://reactos.org/forum/) &bull;
+[Unofficial Community Discord](https://discord.gg/7knjvhT) &bull;
```

Comment:
Either just "Community Discord" or "Community-endorsed Discord" please. Personally I'd just prefer the former.

Reply:
I will fix it